tox.ini

```diff
@@ -25,9 +25,9 @@ deps =
     pytest >= 6.0  # Require >= 6.0 for pyproject.toml support (PEP 517)
     pytest-mock
     pytest-print
-    pytest-profiling
     pytest-sugar
     py # needed for pytest-sugar as it doesn't declare dependency on it.
+    setuptools
```

Comment:
Why is this needed? Is anything relying on `pkg_resources`? If yes, it's best to migrate to `importlib.metadata` instead.

Reply:
I migrated to `importlib.resources` as suggested in: https://importlib-resources.readthedocs.io/en/latest/migration.html#pkg-resources-resource-filename (there is a conditional import of [backport](https://github.com/python/importlib_resources) for Python 3.8).